src/App.tsx

```diff
@@ -1,15 +1,80 @@
-import React from 'react';
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
 import './App.scss';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
+import debounce from 'lodash.debounce';
+
+function filterPerson(arr: Person[], query: string) {
```

Comment:
```suggestion
function filterPeople(people: Person[], query: string) {
```
arr is a bad name for variable/parameter - you should never use it